CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Snyk Vulnerability Scanner Changelog
 
-## [Unreleased]
+## [2.0.1]
```

Comment:
Version numbers are something that we need to maintain manually for now?

Reply:
yes for now. https://github.com/JetBrains/intellij-platform-plugin-template uses it, so I don't want to change official way to handle changelog.
p.s.: perhaps later we'll add some automation.